sys-fs/tmsu/tmsu-0.7.4.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Could you rebase so we get the latest and freshest CI checks on this? Also, golang-build supports EAPI-7 now.